src/ui/views/menu.coffee

```diff
@@ -1,74 +1,168 @@
 remote = require('electron').remote
 Menu = remote.Menu
 
-templateOsx = (viewstate) -> [{
-    label: 'YakYak'
-    submenu: [
-        { label: 'About YakYak', selector: 'orderFrontStandardAboutPanel:' }
-        { type: 'separator' }
-        # { label: 'Preferences...', accelerator: 'Command+,', click: => delegate.openConfig() }
-        { type: 'separator' }
-        { label: 'Hide YakYak', accelerator: 'Command+H', selector: 'hide:' }
-        { label: 'Hide Others', accelerator: 'Command+Shift+H', selector: 'hideOtherApplications:' }
-        { label: 'Show All', selector: 'unhideAllApplications:' }
-        { type: 'separator' }
-        { label: 'Open Inspector', accelerator: 'Command+Alt+I', click: -> action 'devtools' }
+acceleratorMap = {
+    hideyakyak:           {others: 'Control+H', darwin:'Command+H'}
```

Comment:
I don't like putting things in columns like that. I know it kind of makes things more readable but 
- it makes it more difficult to edit things if you use more basic editors (i know there are ways..)
- it tends to make all your code wider which leads to trying to do strange things to avoid too long lines
- when you change something optimal column width may change and you have to tidy it up

Anyway.. it's no a problem. I can live with it, this is just a comment